src/syntax_highlight/null.rs

```diff
@@ -0,0 +1,170 @@
+use liquid;
+use liquid::Token::{self, Identifier};
+use liquid::lexer::Element::{self, Expression, Tag, Raw};
+
+use pulldown_cmark as cmark;
+
+use error;
+
+pub fn has_syntax_theme(_name: &str) -> error::Result<bool> {
+    bail!("Themes are unsupported in this build.");
+}
+
+pub fn list_syntax_themes<'a>() -> Vec<&'a String> {
+    vec![]
+}
+
+pub fn list_syntaxes<'a>() -> Vec<String> {
+    vec![]
+}
+
```

Comment:
I make a distinction between a) no syntaxes/themes due to unsupported feature vs b) no syntaxes/themes due to misconfiguration, though cobalt isn't capable of (b), yet, unless syntect changes/breaks something with the defaults it loads.  This distinction may not be necessary, so I'm interested to see how the null "object" works out.